source/tomopy/prep/normalize.py

```diff
@@ -100,7 +100,8 @@ def minus_log(arr, ncore=None, out=None):
     return out
 
 
-def normalize(arr, flat, dark, cutoff=None, ncore=None, out=None):
+def normalize(arr, flat, dark, averaging='mean',
+              cutoff=None, ncore=None, out=None):
```

Comment:
New parameters need to go at the end of the parameter list; otherwise it breaks function calls that do not use keywords.

```suggestion
def normalize(arr, flat, dark, cutoff=None,
              ncore=None, out=None, averaging='mean'):
```